tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
+    "downlevelIteration": true,
```

Comment:
Forgot to mention, this is to be able to iterate over Iterable data structures (Array-like). Without this, we would have to convert them to arrays using `Array.from(...)`